### PR TITLE
fix: set the correct currency in the alarm description

### PR DESCRIPTION
**Pull request by abarmawi**
alarm description has set static `US$` regardless of the defined currency

## what
* alarm description has set static `US$` regardless of the defined currency

### CODE REVIEW

**Review by diego-ojeda-binbash — APPROVED** [review]

**Comment by exequielrafaela** [comment]
@diego-ojeda-binbash @lgallard I think the PR has a good point and also it looks good . I'll wait for your approval to merge. Thanks in advance.

**Pull request merged**
